README.md

```diff
@@ -175,30 +175,3 @@ Additional information can be found in the [Contributing Readme](https://github.
 ## Code of Conduct
 ArviZ wishes to maintain a positive community. Additional details
 can be found in the [Code of Conduct](https://github.com/arviz-devs/arviz/blob/master/CODE_OF_CONDUCT.MD)
-
-
-### Developing
-
-A typical development workflow is:
```

Comment:
Why remove this section of the doc?

Reply:
Ii is only a really short summary of CONTRIBUTING, therefore I thought it was better to remove it and leave only the link to CONTRIBUTING, otherwise, whenever we update the contributing guide this section has to be updated too. Moreover, I don't think this short summary was complete enough for new contributors (as opposed to the contributing guide).

Reply:
You've convinced me. All sound like great reasons. Thanks for putting the thought into this